lib/pure/concurrency/atomics.nim

```diff
@@ -288,10 +288,15 @@ else:
     type
       # Atomic* {.importcpp: "_Atomic('0)".} [T] = object
 
-      AtomicInt8 {.importc: "_Atomic NI8", size: 1.} = object
-      AtomicInt16 {.importc: "_Atomic NI16", size: 2.} = object
-      AtomicInt32 {.importc: "_Atomic NI32", size: 4.} = object
-      AtomicInt64 {.importc: "_Atomic NI64", size: 8.} = object
+      AtomicInt8Impl {.importc: "_Atomic NI8".} = int8
+      AtomicInt16Impl {.importc: "_Atomic NI16".} = int16
+      AtomicInt32Impl {.importc: "_Atomic NI32".} = int32
+      AtomicInt64Impl {.importc: "_Atomic NI64".} = int64
+
+      AtomicInt8 = distinct AtomicInt8Impl
```

Comment:
```suggestion
      AtomicInt8 {.importc: "_Atomic NI8".} = distinct int8
```

etc.

Reply:
it doesn't work.

Reply:
Hmm. I need to think about this, but it looks to me we either need to make this work or officially embrace "importc type aliases".